### PR TITLE
fix: add new VIRTIO_IMG env varible to operator

### PR DESCRIPTION
**Pull request by ksimon1**
**What this PR does / why we need it**:
fix: add new VIRTIO_IMG env varible to operator
with this new env variable, we will be able to use newest virtio container img available in example pipelines

**Which issue(s) this PR fixes**:
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=2155796

**Special notes for your reviewer**:

**Release note**:
```
NONE
```

### CODE REVIEW

File: pkg/environment/environment.go [hunk 1]
```diff
@@ -43,6 +44,7 @@ const (
 	DeafultCopyTemplateIMG      = "quay.io/kubevirt/tekton-task-copy-template:" + operands.TektonTasksVersion
 	DeafultCleanupVMIMG         = "quay.io/kubevirt/tekton-task-execute-in-vm:" + operands.TektonTasksVersion
 	GenerateSSHKeysIMG          = "quay.io/kubevirt/tekton-task-generate-ssh-keys:" + operands.TektonTasksVersion
+	DefaultVirtioIMG            = "quay.io/kubevirt/virtio-container-disk:20230119_27f97a427"
```
Comment (0xFelix) [inline pkg/environment/environment.go:47]: I'd set that to tag `latest`. But please note that the virtio container disk is not updated very often. Although there are many tags of recent builds it was last changed in Apr 22.

See https://github.com/kubevirt/kubevirt/pull/7546

Comment (ksimon1) [reply pkg/environment/environment.go:47]: I would rather set tag of kubevirt (in this case v0.58.0).

Comment (0xFelix) [reply pkg/environment/environment.go:47]: How do we make sure it is updated and pointing to the matching kubevirt release?

Comment (ksimon1) [reply pkg/environment/environment.go:47]: We can do the same github actions script as we are using in common templates

Comment (0xFelix) [reply pkg/environment/environment.go:47]: You mean an action to update the version periodically in a PR so we have fixed versions in a release? I'd be okay with that.

Comment (ksimon1) [reply pkg/environment/environment.go:47]: yes, that is exactly what I was thinking of.

Comment (0xFelix) [reply pkg/environment/environment.go]: Please do that in a follow up PR.

**Review by 0xFelix — APPROVED** [review]
/lgtm

**Pull request merged**